experiments/tf_trainer/common/tfrecord_input.py

```diff
@@ -7,45 +7,35 @@
 import multiprocessing
 
 import tensorflow as tf
+from tf_trainer.common import base_model
 from tf_trainer.common import dataset_input
+from tf_trainer.common import tfrecord_simple
 from tf_trainer.common import types
+
 from typing import Callable, Dict, List
 
 
-class TFRecordInput(dataset_input.DatasetInput):
+class TFRecordInput(tfrecord_simple.TFSimpleRecordInput):
```

Comment:
I am not a fan of the naming, as it does not highlight enough the difference between both classses.
What about:
TFSimpleRecordInput --> TFRecordInput?
TFRecordInput --> TFRecordInputWithTokenizer?

Reply:
Let's do this in a separate, rename only change.